ts/modules/networkgraph/networkgraph.src.ts

```diff
@@ -0,0 +1,1471 @@
+/* *
+ *
+ *  Networkgraph series
+ *
+ *  (c) 2010-2019 Paweł Fus
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import H from '../../parts/Globals.js';
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface NetworkgraphChart extends DragNodesChart {
+            graphLayoutsLookup: Array<NetworkgraphLayout>;
+            graphLayoutsStorage: Dictionary<NetworkgraphLayout>;
+        }
+        interface NetworkgraphDataLabelsFormatterCallbackFunction {
+            (this: (
+                NetworkgraphDataLabelsFormatterContextObject|
+                DataLabelsFormatterContextObject
+            )): (number|string|null|undefined);
+        }
+        interface NetworkgraphDataLabelsFormatterContextObject
+            extends DataLabelsFormatterContextObject
+        {
+            color: ColorString;
+            key: string;
+            point: NetworkgraphPoint;
+        }
+        interface NetworkgraphDataLabelsOptionsObject
+            extends DataLabelsOptionsObject
+        {
+            format?: string;
+            formatter?: NetworkgraphDataLabelsFormatterCallbackFunction;
+            linkFormat?: string;
+            linkFormatter?: NetworkgraphDataLabelsFormatterCallbackFunction;
+            linkTextPath?: DataLabelsTextPathOptionsObject;
+        }
+        interface NetworkgraphPointMarkerOptionsObject
+            extends PointMarkerOptionsObject
+        {
+            states: NetworkgraphPointStatesOptionsObject;
+        }
+        interface NetworkgraphPointOptions
+            extends LinePointOptions, NodesPointOptions
+        {
+            color?: ColorString;
+            colorIndex?: number;
+            dashStyle?: string;
+            mass?: number;
+            name?: string;
+            opacity?: number;
+            width?: number;
+        }
+        interface NetworkgraphPointStatesInactiveOptionsObject
+            extends PointStatesInactiveOptionsObject
+        {
+            animation?: (boolean|AnimationOptionsObject);
+        }
+        interface NetworkgraphPointStatesOptionsObject
+            extends PointStatesOptionsObject
+        {
+            inactive?: NetworkgraphPointStatesInactiveOptionsObject;
+        }
+        interface NetworkgraphSeriesOptions
+            extends LineSeriesOptions, NodesSeriesOptions
+        {
+            dataLabels?: NetworkgraphDataLabelsOptionsObject;
+            draggable?: boolean;
+            link?: SVGAttributes;
+            marker?: NetworkgraphPointMarkerOptionsObject;
+            mass?: NodesSeriesOptions['mass'];
```

Comment:
There shouldn't be `series.mass` option. Only `series.nodes[index].mass`

Reply:
I removed the mass property. Please review.

Reply:
Thanks 👍